src/app/components/planner-list/planner-list.component.ts

```diff
@@ -251,7 +256,20 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
       //this.treeList.update();
       this.prevWorkItemLength = this.workItems.length;
     }
+    if(this.toolbarHeight) {
+      let toolbarHt:any =  this.toolbarHeight.nativeElement.offsetHeight;
```

Comment:
You could use `number` instead of `any`. We know `offsetHeight` will always be a `number` :)